pkg/kubelet/cm/cpumanager/state/state_checkpoint_test.go

```diff
@@ -227,8 +227,8 @@ func TestCheckpointStateRestore(t *testing.T) {
 			restoredState, err := NewCheckpointState(testingDir, testingCheckpoint, tc.policyName, tc.initialContainers)
 			if err != nil {
 				if strings.TrimSpace(tc.expectedError) != "" {
-					if strings.Contains(err.Error(), "could not restore state from checkpoint") &&
-						strings.Contains(err.Error(), tc.expectedError) {
+					tc.expectedError = "could not restore state from checkpoint: " + tc.expectedError
+					if strings.HasPrefix(err.Error(), tc.expectedError) {
```

Comment:
This change seems unrelated and a bit sstrange to be updating `tc.expectedError` like this. What are you trying to achieve?

Reply:
done